airbyte-integrations/bases/base-python/base_python/catalog_helpers.py

```diff
@@ -32,7 +32,7 @@ def coerce_catalog_as_full_refresh(catalog: AirbyteCatalog) -> AirbyteCatalog:
         """
         coerced_catalog = catalog.copy()
         for stream in catalog.streams:
-            stream.source_defined_cursor = False
+            stream.cursor = False
```

Comment:
this is a named field in the airbyte protocol -- `AirbyteStream.cursor` doesn't exist

Reply:
whoops. accidental refactor.